pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -248,20 +248,22 @@ func (cs *ConntrackConnectionStore) AddOrUpdateConn(conn *flowexporter.Connectio
 			conn.StopTime = time.Now()
 		}
 		metrics.TotalAntreaConnectionsInConnTrackTable.Inc()
-		klog.V(4).Infof("New Antrea flow added: %v", conn)
-		// Add new antrea connection to connection store
+		conn.IsActive = true
+		conn.IsDenyConn = false
+		// Add new antrea connection to connection store and PQ.
 		cs.connections[connKey] = conn
+		cs.connectionStore.addItemToQueue(connKey, conn)
+		klog.V(4).Infof("New Antrea flow added: %v", conn)
 	}
 }
 
-// DeleteConnWithoutLock deletes the connection from the connection map given
-// the connection key without grabbing the lock. Caller is expected to grab lock.
-func (cs *ConntrackConnectionStore) DeleteConnWithoutLock(connKey flowexporter.ConnectionKey) error {
-	_, exists := cs.connections[connKey]
-	if !exists {
-		return fmt.Errorf("connection with key %v doesn't exist in map", connKey)
+// checkConnActiveness returns true if there are changes in connection's stats or
+// TCP state, representing the connection is active.
+func (cs *ConntrackConnectionStore) checkConnActiveness(conn, existingConn *flowexporter.Connection) bool {
```

Comment:
we can name this function to `isConnActive`

Reply:
changed to `checkConnActive`, slightly feel `isConnActive` sounds more like the naming of a boolean field. I can definitely change to `isConnActive` if that's only me.